CMakeLists.txt

```diff
@@ -1,88 +1,218 @@
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.4.1)
-project(kompute VERSION 0.8.1)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
Strictly speaking 3.14 would be the minimum requirement: https://cmake.org/cmake/help/latest/module/FetchContent.html?highlight=fetchcontent_declare#command:fetchcontent_makeavailable

I will downgrade it to 3.14.

Reply:
Ok tht's good to know, we can update in the documentation as well so people are aware, this fetchcontent would be the main one to make sure this would work, good shout